packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/CameraPlugin.swift

```diff
@@ -0,0 +1,520 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+import ObjectiveC
+
+// Import Objectice-C part of the implementation when SwiftPM is used.
+#if canImport(camera_avfoundation_objc)
+  import camera_avfoundation_objc
+#endif
+
+public typealias CaptureNamedDeviceFactory = (String) -> FLTCaptureDevice
```

Comment:
why is it public?

Reply:
It was public because `init` (not the `convenience init`) was public but neither has to be